### DISCUSSION TITLE
关于扩展的设计与建议征集

### DISCUSSION THREAD

**Discussion by alexcui03** [Clipteam/clipcc-gui#60] [闲聊/General]
根据目前的更新规划，3.1 版本已基本确定为更新并完善扩展功能。现对于扩展征集相关建议。

扩展的文档在 [clipcc-extension](https://github.com/clipteam/clipcc-extension) 仓库的 `std/v1-draft.md` 文件中，该文件可能随时更新。

欢迎各位积极提出建议！

**Comment by SimonShiki** [comment]
文档还没上传吗

**Reply by alexcui03** [reply]
草 我没push

**Reply by SimonShiki** [reply]
gkd
